package.json

```diff
@@ -4,19 +4,26 @@
   "private": true,
   "dependencies": {
     "@contentful/forma-36-fcss": "0.0.34",
-    "@contentful/forma-36-react-components": "^3.28.7",
+    "@contentful/forma-36-react-components": "^3.42.2",
+    "@contentful/forma-36-tokens": "^0.5.2",
+    "@contentful/rich-text-html-renderer": "^14.1.1",
+    "@contentful/rich-text-react-renderer": "^14.1.1",
```

Comment:
please remove since we're not using it

Reply:
only "@contentful/forma-36-tokens": "^0.5.2",
and "@contentful/rich-text-react-renderer": "^14.1.1", right?
or all 4 above?